test/parallel/test-crypto-dh-stateless.js

```diff
@@ -12,6 +12,19 @@ assert.throws(() => crypto.diffieHellman(), {
   message: 'The "options" argument must be of type object. Received undefined'
 });
 
+assert.throws(() => crypto.diffieHellman(null), {
+  name: 'TypeError',
+  code: 'ERR_INVALID_ARG_TYPE',
+  message: 'The "options" argument must be of type object. Received null'
+});
+
+assert.throws(() => crypto.diffieHellman([]), {
+  name: 'TypeError',
+  code: 'ERR_INVALID_ARG_TYPE',
+  /* eslint-disable max-len */
```

Comment:
Instead of turning off eslint, can you split the message up across lines.

Reply:
I have tried to split it, but the length is still more than 80.

Reply:
Check out how the keys are split across lines in this same file: https://github.com/nodejs/node/blob/feb1519b99080fd0fdd78437b03c0f7365797809/test/parallel/test-crypto-dh-stateless.js#L61-L71